automated install/basic-install.sh

```diff
@@ -414,6 +414,8 @@ make_repo() {
     fi
     # Clone the repo and return the return code from this command
     git clone -q --depth 1 "${remoteRepo}" "${directory}" &> /dev/null || return $?
+    # Data in the repositories is public anyway so we can make it readable by everyone (+r to keep executable permission if already set by git)
+    chmod -R a+r "${directory}"
```

Comment:
Is `umask` affecting git clone? If not, then this is not necessary.

Reply:
I don't know if `umask` is the reason or some configuration setting in `git` (e.g. [core.sharedRepository ](https://git-scm.com/docs/git-config#Documentation/git-config.txt-coresharedRepository), which I have not set), but when I do:
```
pi@pi-hole:~/Downloads $ git clone --depth 1 https://github.com/pi-hole/pi-hole.git pihole
Klone nach 'pihole' ...
remote: Enumerating objects: 81, done.
remote: Counting objects: 100% (81/81), done.
remote: Compressing objects: 100% (73/73), done.
remote: Total 81 (delta 6), reused 30 (delta 0), pack-reused 0
Entpacke Objekte: 100% (81/81), Fertig.
pi@pi-hole:~/Downloads $ cd pihole
pi@pi-hole:~/Downloads/pihole $ ls -la
insgesamt 160
drwxr-x--- 10 pi pi  4096 Mai  4 11:29 .
drwxr-xr-x  4 pi pi  4096 Mai  4 11:29 ..
drwxr-x---  6 pi pi  4096 Mai  4 11:29 advanced
drwxr-x---  2 pi pi  4096 Mai  4 11:29 automated install
-rwxr-x---  1 pi pi    20 Mai  4 11:29 autotest
drwxr-x---  2 pi pi  4096 Mai  4 11:29 block hulu ads
-rw-r-----  1 pi pi  2489 Mai  4 11:29 CONTRIBUTING.md
-rw-r-----  1 pi pi   777 Mai  4 11:29 .editorconfig
drwxr-x---  8 pi pi  4096 Mai  4 11:29 .git
-rw-r-----  1 pi pi   688 Mai  4 11:29 .gitattributes
drwxr-x---  2 pi pi  4096 Mai  4 11:29 .github
-rw-r-----  1 pi pi  1385 Mai  4 11:29 .gitignore
-rwxr-x---  1 pi pi 25299 Mai  4 11:29 gravity.sh
drwxr-x---  3 pi pi  4096 Mai  4 11:29 .idea
-rw-r-----  1 pi pi 14204 Mai  4 11:29 LICENSE
drwxr-x---  2 pi pi  4096 Mai  4 11:29 manpages
-rwxr-x---  1 pi pi 14637 Mai  4 11:29 pihole
-rw-r-----  1 pi pi 17298 Mai  4 11:29 README.md
-rw-r-----  1 pi pi    60 Mai  4 11:29 requirements.txt
-rw-r-----  1 pi pi   107 Mai  4 11:29 setup.py
-rw-r-----  1 pi pi    69 Mai  4 11:29 .stickler.yml
drwxr-x---  2 pi pi  4096 Mai  4 11:29 test
-rw-r-----  1 pi pi   384 Mai  4 11:29 tox.ini
-rw-r-----  1 pi pi   211 Mai  4 11:29 .travis.yml
```
because the [documentation](https://git-scm.com/docs/git-clone) says

> Clones a repository into a newly created directory, creates remote-tracking branches for each branch in the cloned repository (visible using git branch -r), and creates and **checks out** an initial branch that is forked from the cloned repository’s currently active branch.

So I would suppose, that `git clone` does a `git checkout` in the background and then respects the users `umask` if not configured otherwise.

Reply:
As long as `git status` doesn't show the files as changed, then this is fine.

Reply:
After a quick test, it looks like Git is fine with this.